src/test/java/com/org/jenkins/custom/jenkins/distribution/service/PluginControllerTest.java

```diff
@@ -29,23 +31,26 @@ public void setup() {
     @Test
     public void return_status_200() throws Exception {
 
+        MockMvc pluginMvc = MockMvcBuilders.webAppContextSetup(webApplicationContext).build();
```

Comment:
Maven was returning null on running these tests since the web app context was not available in the @Before annotation, so injected it in the test.